src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -2250,10 +2251,7 @@ private void updateFromRoomConfiguration()
 
     private void processRoomMetadataJson(String json)
```

Comment:
The catch block used to span the entire method, now it only covers the json parser. Is this intended and safe?

Reply:
yep. I have added one default value that was missing. So it should be safe now.

Reply:
there's a bunch of other possible exceptions here, for example "startMuted: 0" will cause the cast to fail